ports/kf5iconthemes/vcpkg.json

```
@@ -0,0 +1,40 @@
+{
+  "name": "kf5iconthemes",
+  "version-semver": "5.84.0",
```

```suggestion
  "version": "5.84.0",
```
Considering the discussion here https://github.com/microsoft/vcpkg/pull/19671#discussion_r704696052, it might be more suitable to use `version `here. Sorry for the confusion about this before.

---

@NancyLi1013 no worries, we're all learning here as we go. Fixed it.